### DISCUSSION TITLE
关于IC2中在<各类机器>内使用<单元类物品>时不会返还<空单元>bug

### DISCUSSION THREAD

**Discussion by WaterDouble** [IndustrialCraftMC/IndustrialCraft2#34] [【漏洞反馈】Bugs Report]
使用模组版本：1.industrialcraft-2-2.9.84+ex119-1.19.2-forge
2.[1.19.2]Jei物品管理器(forge11.3.0.262)

游戏版本：1.19.2 forge-43.2.0 Op-I1-pre4

问题描述：在游玩时发现在<各类机器>内使用<单元类物品>时不会返还<空单元>。

**Comment by WaterDouble** [comment]
直接使用<岩浆单元>和<水单元>可以正常倒出<岩浆>和<水>。

**Comment by AbbottMc** [comment]
该问题已有相关反馈，见：
[> 在地热发电机内放入熔岩单元，没有返还空单元 ![image](https://user-images.githubusercontent.com/16969459/236326187-bb387401-8914-4e69-bf20-9eb7d12ad575.png)
](https://github.com/AbbottMc/IndustrialCraft2/discussions/14#discussion-5161422)
